packages/evm/src/opcodes/gas.ts

```diff
@@ -521,7 +524,7 @@ export const dynamicGasHandlers: Map<number, AsyncDynamicGasHandler | SyncDynami
         if (value > BigInt(0)) {
           gas += common.param('gasPrices', 'authcallValueTransfer')
           const account = await runState.eei.getAccount(toAddress)
-          if (account.isEmpty()) {
+          if (!account) {
```

Comment:
Can't 100% answer this in the context to be honest. Again, regarding that `getAccount()` is now returning `undefined` and not a combined empty account for (undefined || empty) I would assume for this situation that `undefined` is meant since the gas price param is also called `callNewAccount`.

Does this make sense?

Reply:
Oh right good point. I think the new version is actually better than the old version :thinking:

Reply:
That was the intent at least. 🙂

Reply:
Guess we will stumble upon several occasions where we have to (and are now able to) re-answer this same or somewhat adopted question: what was actually meant? Undefined or empty? (which should be a good thing)